utilities/post_migration.py

```
@@ -399,32 +399,136 @@ def _verify_gateway(interface_name: str, expected_gateway: str, matching_interfa
         )
 
 
+def _load_current_interfaces(
+    ssh_conn: Any,
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `ssh_conn`

(ANN401)